cairo/tests/ethereum/crypto/test_bls12_381.py

```diff
@@ -197,15 +212,25 @@ def test_blsp2_add_negated_y(self, cairo_run, p: Optimized_Point3D[BLSF2]):
         def test_blsp2_add(
             self, cairo_run, p: Optimized_Point3D[BLSF2], q: Optimized_Point3D[BLSF2]
         ):
-            assert cairo_run("blsp2_add", p, q) == normalize1(add(p, q))
+            expected = add(p, q)
+            if not is_inf(expected):
+                expected = normalize1(expected)
+            assert cairo_run("blsp2_add", p, q) == expected
 
         @given(p=...)
         def test_blsp2_double(self, cairo_run, p: Optimized_Point3D[BLSF2]):
-            assert cairo_run("blsp2_double", p) == normalize1(double(p))
+            expected = double(p)
+            if is_inf(expected):
+                expected = Z2
+            else:
+                expected = normalize1(expected)
```

Comment:
Is this necessary?
shouldn't this be done in serde in some way?
```suggestion
            p_double = double(p)
            expected = Z2 if is_inf(p_double) else normalize1(p_double)
```

Reply:
We're making modifs on the `py_ecc` expected result, so it won't be handled by args_gen/serde, right?
(args_gen does `p`in python --> `p` in Cairo, and serde does `result` in Cairo --> `result` in python ; to compare `result` and `expected`)